db-acra-migration/python/migration.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2016, Cossack Labs Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# coding: utf-8
+import argparse
+import io
+
+from sqlalchemy import (Table, Column, Integer, MetaData, select, update, Text, func, asc,)
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.dialects import postgresql
+
+from common import get_engine, register_common_cli_params
+
+metadata = MetaData()
+user_table = Table(
+    'users', metadata,
+    Column('id', Integer, primary_key=True, nullable=False),
+    Column('phone_number', Text, nullable=False),
+    Column('ssn', Text, nullable=False),
+    Column('email', Text, nullable=False),
+    Column('firstname', Text, nullable=False),
+    Column('lastname', Text, nullable=False),
+    Column('age', Integer, nullable=False),
+)
+
+
+def generate_data(engine, csv_string):
+    session = sessionmaker(bind=engine)()
```

Comment:
if we use sessions lets use it in the [convenient way](https://docs.sqlalchemy.org/en/20/orm/session_basics.html):
```
from sqlalchemy.orm import Session
...
...
with Session(engine) as session:
  with session.connection() as connection:
    with connection.cursor() as cursor():
       #do something
  
```